BSP/Inc/BSP_CAN.h

```diff
@@ -15,13 +15,13 @@
  */
 void BSP_CAN_Init(callback_t rxEvent, callback_t txEnd, bool loopback);
 
+
 /**
- * @brief   Transmits the data onto the CAN bus with the specified id
- * @param   id : Message of ID. Also indicates the priority of message. The lower the value, the higher the priority.
- * @param   data : data to be transmitted. The max is 8 bytes.
- * @param   length : num of bytes of data to be transmitted. This must be <= 8 bytes or else the rest of the message is dropped.
- * @return  ERROR if module was unable to transmit the data onto the CAN bus. SUCCESS indicates data was transmitted.
+ * @brief   Calls stm-level CAN_DeInit
+ * @return  None
  */
+void BSP_CAN_DeInit();
+
```

Comment:
You should have comments for both functions.

Reply:
which functions? BSP_CAN_DeInit has them